proofs/tree_test.go

```diff
@@ -1566,3 +1566,25 @@ func Test_Enums(t *testing.T) {
 		assert.True(t, valid, "proof must be valid")
 	}
 }
+
+func Test_integers(t *testing.T) {
+	doc := new(documentspb.Integers)
+	salts := new(documentspb.SaltedIntegers)
+	FillSalts(doc, salts)
+
+	doctree := NewDocumentTree(TreeOptions{Hash: sha256.New()})
+	doctree.AddLeavesFromDocument(doc, salts)
+	doctree.Generate()
+	fmt.Printf("Generated tree: %s\n", doctree.String())
+
+	for _, field := range []string{"valueA", "valueB", "valueG", "valueH", "valueJ"} {
```

Comment:
Do we need an example with uintX ?

Reply:
It was failing at flattening itself. The proofs are just some random ones I took ;)